packages/engine/crawlers/solana/index.ts

```diff
@@ -56,10 +59,14 @@ export const solanaEngineRunner: SolanaRunner = {
 				item.account.quotes = quotes[makeHashId(item)].quotes;
 			}
 
-			tokenActions.setItems(fungibleTokens);
+			tokenActions.setTokens(fungibleTokens);
+			solanaCollectiblesByAddress({
+				context,
+				address: currentPubkey.toString(),
+			});
 
 			for (const key of accountKeys) {
-				registerAccountChanges(context, liveConnection, key);
+				registerAccountChanges(context, liveConnection, key, currentPubkey);
```

Comment:
with passing the context, consider to remove `liveConnection`, this connection changes the commitment for request, we could reconstruct the new connection each time we need (it might not take performance effect)

Reply:
I found the problem of `liveConnection`, It seems like we missunderstood about commitment, for listen account change faster on solana, we use confirm commitment. But instead of reconstruct the connection, we could add optional argument `commitment` when listen to account change. So pls refactor this one by removing live connection, and change any liveConnection using to add `confirm` in the listening function call.